{"operations":[{"i":-1,"o":1,"p":{"01621fb84e320000f001":"public","01692aa71f3108454479":true,"01703376c4a81c0e9d94":"stable","fdff023000000000000011":"Catch Error Type","ff023000000000000011":"Catch Error Type","v":2},"t":"016eeb5ede9f3094d900"},{"i":-1,"o":1,"p":{"01740c5d4ae4b520dd01":2,"ff023000000000000011":"Message","v":1},"t":"016eeb5ede9f3094d903"},{"i":-1,"o":1,"p":{"01740c5d4ae4b520dd01":0,"ff023000000000000011":"Control Flow","v":1},"t":"016eeb5ede9f3094d904"},{"i":-1,"o":1,"p":{"01740c5d4ae4b520dd01":1,"ff023000000000000011":"Error Flow","v":1},"t":"016eeb5ede9f3094d905"},{"i":-1,"o":1,"p":{"01740c5d4ae4b520dd01":0,"ff023000000000000011":"Error Flow","v":1},"t":"016eeb5ede9f3094d902"},{"i":-1,"o":1,"p":{"01703376c4a81c0e9d91":"Catches errors and extract the details.","01703376c4a81c0e9d92":"Catching an error from an error-flow, and if the given code is the same than the error code, it allows you to access its details and trigger a control-flow. If the codes are different, it simply forward the error-flow.","v":1},"t":"017b35893ba3d434598c"},{"i":-1,"o":1,"p":{"01740c5d4ae4b520dd01":1,"ff023000000000000011":"Code","v":1},"t":"016eeb5ede9f3094d901"},{"i":-1,"o":1,"p":{"01703376c4a81c0e9d92":"The error message.","v":1},"t":"017b3589429ca087929a"},{"i":-1,"o":1,"p":{"01703376c4a81c0e9d92":"The output control flow.","v":1},"t":"017b358942a250406542"},{"i":-1,"o":1,"p":{"01703376c4a81c0e9d92":"The input error flow, if the input code and the error code are different.","v":1},"t":"017b358942a34ead89c5"},{"i":-1,"o":1,"p":{"01703376c4a81c0e9d92":"The error flow from which to catch errors.","v":1},"t":"017b35893fb22a73d569"},{"i":-1,"o":1,"p":{"01703376c4a81c0e9d92":"The error code that is accepted.","v":1},"t":"017b35893fb247078206"}],"optionalOperations":[],"rootTag":"016eeb5ede9f3094d900"}
